src/aap_eda/core/validators.py

```diff
@@ -68,3 +69,14 @@ def check_awx_tokens(user_id: int) -> int:
         )
 
     return user_id
+
+
+def is_extra_var_dict(extra_var: str):
+    try:
+        data = yaml.safe_load(extra_var)
+        if not isinstance(data, dict):
+            raise serializers.ValidationError("yaml is not in object format")
```

Comment:
It might be a json too, I think it is more accurate and consistent with the other validation error: 
```suggestion
            raise serializers.ValidationError("Extra var is not in object format")
```